### ISSUE TITLE
Amounts wrapping around!

### ISSUE THREAD

**Issue by xenog** [haskoin/haskoin-wallet#103]
```
xeno@carbonite:~/Source/Haskoin/haskoin-wallet$ hw -t -R send satoshi mfcyDmadWwy85woDcj2u6uttfn7VMhSEq7 1000
hw: buildTx: Invalid amount 18446744073709542616
```

**Comment by plaprade** [comment]
https://github.com/haskoin/haskoin-wallet/blob/master/Network/Haskoin/Wallet/Transaction.hs#L1186

**Comment by xenog** [comment]
Fixed in pull request #102.
